src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -11,6 +13,16 @@ namespace Microsoft.Extensions.Logging
 {
     public static class ILoggerExtensions
     {
+        private const string RequestFormat =
```

Comment:
I think lots of perf improvement here when we only have to create once this format.

Reply:
Definately makes sense, thanks!